app/javascript/dashboard/components/widgets/conversation/contextMenu/menuItemWithSubmenu.vue

```diff
@@ -83,7 +83,7 @@ export default {
   }
 
   &.disabled {
-    opacity: 50%;
+    opacity: 0.5;
```

Comment:
% opacity gets converted to 0.1 

Related: https://github.com/NMFR/optimize-css-assets-webpack-plugin/issues/118

We might need to upgrade the dependancies.